helm/charts/determined/templates/genai/genai-initialize-shared-fs-permissions.yaml

```diff
@@ -48,12 +48,10 @@ spec:
           - bash
           - -exc
           - |
-            apt-get update -y && apt-get install acl -y;
             echo "whoami: $(whoami)";
             chmod 2775 /shared_fs;
             GROUP_ID={{ (required "A valid .Values.genai.agentGroupID entry required!" .Values.genai.agentGroupID) }};
             chgrp +${GROUP_ID} /shared_fs;
-            setfacl -d -m g::rwX /shared_fs;
```

Comment:
Note: We don't need this and it also won't work on all types of shared drives as tyler and I found in smoke testing. This is solved properly in genai by asserting a umask on all file operations.